datumaro/plugins/coco_format/converter.py

```diff
@@ -520,13 +520,27 @@ def _split_tasks_string(s):
     @classmethod
     def build_cmdline_parser(cls, **kwargs):
         kwargs['description'] = """
-            Segmentation mask modes (--segmentation-mode):|n
+            Segmentation mask modes ('--segmentation-mode'):|n
             - '{sm.guess.name}': guess the mode for each instance,|n
             |s|suse 'is_crowd' attribute as hint|n
             - '{sm.polygons.name}': save polygons,|n
             |s|smerge and convert masks, prefer polygons|n
             - '{sm.mask.name}': save masks,|n
-            |s|smerge and convert polygons, prefer masks
+            |s|smerge and convert polygons, prefer masks|n
+            |n
+            The '--reindex' option allows to control if the images and
+            annotations must be given new indices. It can be useful, when
+            you want to preserve the original indices in the produced dataset.
+            Consider enabling this option when converting from other formats.|n
```

Comment:
The advice to enable this option seems strange when you've changed the default to true. Maybe change it to "keeping this option enabled"?